src/components/common.js

```diff
@@ -275,6 +275,16 @@ export default {
       }
 
       return this.id;
+    },
+    zarrMetadataUrl() {
+      const zarrMetadata = this.assets.find(x =>
+        x.roles.includes("zarr-consolidated-metadata")
```

Comment:
```suggestion
        x.roles.includes("zarr-v2-consolidated-metadata")
```

I think we should use a version-specific attribute here. Zarr v3 will look different.

Reply:
I agree with having version-specific parsing/rendering of the metadata - if we plan on supporting Zarr v3 down the line, we should also figure out how we would like to pass information about the Zarr version down to the `parseMeta` function:

- This computed property could return a version number along with the URL based on the role of the Zarr metadata asset
- The `ZarrMetadataTab` could check the version by accessing the group's `zarr_format` after downloading the file

In either case, this version could be used to call a specified `parseMetaVX`, or throw an error if the version isn't supported.